queries/gnuplot/highlights.scm

```diff
@@ -0,0 +1,632 @@
+
+[
+  "tc"
+  "fc"
+  "fs"
+  "lc"
+  "ls"
+  "lw"
+  "lt"
+  "pt"
+  "ps"
+  "pi"
+  "pn"
+  "dt"
+  "as"
+  "start"
+  "cycles"
+  "saturation"
+  "interval"
+  "format"
+  "keywidth"
+  "samplen"
+  "columns"
+  "title"
+  "notitle"
+  "every"
+  "index"
+  "using"
+  "with"
+  "frac"
+  "cb"
+  "arg"
+  "prefix"
+  "output"
+  "primary"
+  "specular"
+  "spec2"
+  "firstlinetype"
+  "width"
+  "height"
+  "expand"
+  "array"
+  "dx"
+  "dy"
+  "dz"
+  "filetype"
+  "center"
+  "record"
+] @variable.member
+
+[(angles)
+(clip)
+(colorsequence)
+(contour)
+(encoding)
+(mapping)
+(xdata)
+(theta)
+"wall"
+"on"
+"off"
+"opaque"
+"inside"
+"outside"
+"margin"
+"cen"
+"lef"
+"rig"
+"top"
+"bot"
+"lr"
+"a"
+"maxcols"
+"maxrows"
+"autojustify"
+"overlap"
+"spread"
+"wrap"
+"swarm"
+"range"
+"label"
+"mixed"
+"triangles"
+"insidecolor"
+"noinsidecolor"
+"cycle"
+"tics"
+"ztics"
+"cbtics"
+"user"
+"front"
+"back"
+"bdefault"
+"time"
+"palette"
+"terminal"
+"onecolor"
+"invert"
+"reverse"
+"writeback"
+"extend"
+"restore"
+"linear"
+"cubicspline"
+"bspline"
+"points"
+"order"
+"levels"
+"sorted"
+"autofreq"
+"add"
+"inout"
+"axis"
+"mirror"
+"type"
+"rowsfirst"
+"columnsfirst"
+"downwards"
+"upwards"
+"prevnext"
+"gray"
+"color"
+"gamma"
+"defined"
+"cubehelix"
+"model"
+"maxcolors"
+"file"
+"colormap"
+"rgbformulae"
+"viridis"
+"positive"
+"negative"
+"nops_allcF"
+"ps_allcF"
+"quiet"
+"full"
+"trip"
+"numbers"
+"small"
+"large"
+"fullwidth"
+"append"
+"bind"
+"errors"
+"session"
+"behind"
+"polar"
+"layerdefault"
+"locale"
+"axes"
+"fix"
+"keepfix"
+"noextend"
+"head"
+"fixed"
+"filled"
+"nofilled"
+"absolute"
+"at"
+"relative"
+"enhanced"
+"border"
+"noborder"
+"rgbcolor"
+"empty"
+"black"
+"bgnd"
+"nodraw"
+"size"
+"new"
+"first"
+"second"
+"screen"
+"graph"
+"character"
+"trianglepattern"
+"undefined"
+"noundefined"
+"altdiagonal"
+"bentover"
+"vertical"
+"horizontal"
+"square"
+"ratio"
+"noratio"
+"solid"
+"transparent"
+"pattern"
+"from"
+"to_rto"
+"length"
+"angle"
+"columnheaders"
+"fortran"
+"nofpe_trap"
+"missing"
+"separator"
+"commentschars"
+"log"
+"rangelimited"
+"offset"
+"nooffset"
+"scale"
+"font"
+"point"
+"nopoint"
+"boxed"
+"noboxed"
+"hypertext"
+"defaults"
+"keyentry"
+"splines"
+"qnorm"
+"gauss"
+"cauchy"
+"exp"
+"box"
+"hann"
+"implicit"
+"explicit"
+"rotate"
+"by"
+"parallel"
+"norotate"
+"map"
+"projection"
+"equal"
+"azimuth"
+"nohidden3d"
+"nocontours"
+"nosurface"
+"colornames"
+"functions"
+"variables"
+"version"
+"nologfile"
+"logfile"
+"fit_out"
+"errorvariables"
+"covariancevariables"
+"errorscaling"
+"prescale"
+"maxiter"
+"limit"
+"limit_abs"
+"start-lambda"
+"lambda-factor"
+"script"
+"clip"
+"fontscale"
+"lighting"
+"depthorder"
+"interpolate"
+"corners2color"
+"flush"
+"scanorder"
+"hidden3d"
+"clipcb"
+"layout"
+"margins"
+"spacing"
+"smooth"
+"binary"
+"skip"
+"bins"
+"binrange"
+"binwidth"
+"binvalue"
+"mask"
+"convexhull"
+"concavehull"
+"volatile"
+"zsort"
+"nonuniform"
+"sparse"
+"matrix"
+] @attribute
```

Comment:
What happened here? 🤔

Reply:
:shrug: script outputs that

Reply:
@lucario387

Reply:
script trips :)
Too many nodes and it breaks

Reply:
```diff
diff --git a/queries/gnuplot/highlights.scm b/queries/gnuplot/highlights.scm
index 2c66ba06..26f64195 100644
--- a/queries/gnuplot/highlights.scm
+++ b/queries/gnuplot/highlights.scm
@@ -287,6 +287,7 @@
   "record"
 ] @variable.member
 
+; Workaround due to the formatter cannot handle 300 nodes under a list
 [(angles)
 (clip)
 (colorsequence)
@@ -391,6 +392,8 @@
 "fix"
 "keepfix"
 "noextend"
+] @attribute
+[
 "head"
 "fixed"
 "filled"
```

In the mean time you can apply this diff :thinking:

Reply:
Or format it manually and add `; format-ignore` on top.